CONTRIBUTING.md

```diff
@@ -16,8 +16,8 @@ Keep in mind that this repository is for the Wazuh documentation. There are some
 - If you have **problems, bugs or unexpected results** when using any of the Wazuh components, create the issue in its respective repository:
   - [Wazuh Core](https://github.com/wazuh/wazuh/issues)
   - [Wazuh API](https://github.com/wazuh/wazuh-api/issues)
-  - [Wazuh App](https://github.com/wazuh/wazuh-dashboard-plugins/issues)
-  - [Wazuh Splunk app](https://github.com/wazuh/wazuh-splunk/issues)
+  - [Wazuh Indexer](https://github.com/wazuh/wazuh-indexer/issues)
+  - [Wazuh Dashboard](https://github.com/wazuh/wazuh-dashboard/issues)
```

Comment:
**nitpick**: The applications are called `Wazuh indexer` and `Wazuh dashboard`

Reply:
Why are we removing the entries related to the plugins?

Reply:
Our main repositories are going to be wazuh-indexer and wazuh-dashboard. As wazuh-dashboard represents the whole app, is going to be more straightforward for the users to report the bugs.The users may not be able to identify if a bug is happening due to the plugin or the dashboard structure. 
Additionally, we the repositories already exist and we are currently receiving issues through them